All Files Are Working Properly in Feature branch

asadmisrani opened the pull request:
Every code in this branch is working properly

asadmisrani committed:
Added Description

asadmisrani committed:
Html file Added

asadmisrani committed:
Css Style Sheet Added

asadmisrani committed:
All Files uploaded

asadmisrani committed:
error updated

osamaashraf6 commented on .gitignore:
```diff
@@ -0,0 +1,104 @@
+
+# Compiled binary addons (https://nodejs.org/api/addons.html)
+build/Release
+
+# Dependency directories
+node_modules/
+jspm_packages/
+
+# TypeScript v1 declaration files
```
- [x] Great job 👌at adding the node modules to the gitignore file, but I see that you didn't put the ``` test.md``` file inside the gitignore, so kindly add the ``` test.md``` file inside the gitignore.💯
- [x] [OPTIONAL] Also consider cleaning 😃 the gitignore file from all the unnecessary things 😊just put the ```node_modules and the test.md``` inside it and in the future, you can add whatever you want.

osamaashraf6 commented on README.md:
```diff
@@ -1 +1,13 @@
-# microverse.github.io
\ No newline at end of file
+# microverse.github.io
+
+> This is website that used for Microverse
+
+**Built With**
+Html
+Css-3
+Linters
+
```
- [x] Kindly 😊add to your README file, the [template provided by Microverse](https://github.com/microverseinc/readme-template)⤵️ and complete it with your personal information💯. In this way, you will have documented your work in a professional way😃

osamaashraf6 reviewed (changes requested):
Hi @asadmisrani 👋, 

Good job so far! 👏🥳
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

![SoCloseAlmostGIF](https://user-images.githubusercontent.com/83360636/211644747-5c7dcf67-7551-4ed5-8127-3d3a666b79e2.gif)


## Highlights ✔️

- [x] Good usage in correcting GitHub flow💯 
- [x] Good work on the project.



## Required Changes ♻️

- [x] Great job😊 at implementing the linter, but I see that there is an error🐛 at the stylelint linter, so please fix this linter by adding this command at the terminal 
``` npx stylelint "**/*.{css,scss}" --fix``` ,so the linter will be happy for that 😃

- [x]  You did well 🎈🪄on the project, but I see that you didn't write a pull request for this project, so please😃 add a Pull Request which should have a descriptive title telling you what changes are inside and Pull Request should have a summary describing introduced changes in more detail.💯

- [x] Please consider using best practices when you write commit messages, they should be capitalized sentences in the imperative form, for example: "Add or Upload all files " instead of "All Files uploaded]" ... etc 👍

- [x] Good job👍with the project you are almost there you finish the project don't worry 🥳your code is good, but there are some issues to be fixed, so please check the comments to know what I am talking about 😊. 

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

asadmisrani committed:
test md file uploaded

asadmisrani commented:
I did correct all the changes that you sent to me.
Corrected linters error and fix npx stylelint.
Updated Readme file.
I also send test.md file to .gitignore.

MaganaAsiati commented on README.md:
```diff
@@ -1 +1,13 @@
+**Built With**
+Html
+Css-3
+Linters
```
- kindly Add a descriptive README file to your project - please use [this template](https://github.com/microverseinc/readme-template) as required in the project but customize it to your project. as it was required [here](https://github.com/asadmisrani/microverse.github.io/pull/1/files#r1066289861) by the previous reviewer :+1:

MaganaAsiati commented on .gitignore:
```diff
@@ -0,0 +1,104 @@
+# Output of 'npm pack'
+*.tgz
+
+# Yarn Integrity file
+.yarn-integrity
+
+# dotenv environment variables file
+.env
+.env.test
+
+# parcel-bundler cache (https://parceljs.org/)
+.cache
+
+# Next.js build output
+.next
+
+# Nuxt.js build / generate output
+.nuxt
+dist
+
+# Gatsby files
+.cache/
+# Comment in the public line in if your project uses Gatsby and *not* Next.js
+# https://nextjs.org/blog/next-9-1#public-directory-support
+# public
+
+# vuepress build output
+.vuepress/dist
+
+# Serverless directories
+.serverless/
+
+# FuseBox cache
+.fusebox/
+
+# DynamoDB Local files
+.dynamodb/
+
+# TernJS port file
+.tern-port
```
- kindly Remove unnecessary files in your gitignore and make sure that the file listing `test.md` file and `node_modules` directory is added in the gitignore (as it can be added by accident while setting up Linters). as it shows above alone `node_modules` is present in the git ignore please add also` test.md` :+1:

MaganaAsiati reviewed (changes requested):
# STATUS: Required Changes ♻️♻️♻️ 
## Hi @asadmisrani , I'm @MaganaAsiati, your reviewer for this PR  :wave: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

![so-close-gif-4](https://user-images.githubusercontent.com/95297251/191952218-b7185270-c02b-4e4d-89af-eae77fcd8bea.gif)

## Highlights

- Descriptive README file  :heavy_check_mark:👨🏻‍💻
- Great work using the right GitHub Flow.:heavy_check_mark:
- No Linter errors :heavy_check_mark:

## Required Changes ♻️
 
- You did a great job by setting up correctly linters, but as you see it still showing red color at the stylelint linter because you still have linter errors, so please fix this linter by executing this command at the terminal
` npx stylelint "**/*.{css,scss}" --fix ,so the linter will be happy for that :smiley: `

 - Kindly Add a meaningful description to your pull request in the future which gives more information about what you implement in this milestone heavy_check_mark👨🏻‍💻


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

- [ ]Please consider using best practices when you write commit messages, they should be capitalized sentences in the imperative form, for example: "Add or Upload all files " instead of "All Files uploaded]" ... etc 👨🏻‍💻



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

asadmisrani commented:
Hello Sir,
I have already fixed this error npx stylelint "**/*.{css,scss}" --fix can
you please approve my Project then I will go further.
Thank You
Regards
Asad Shareef

On Wed, Jan 11, 2023 at 2:56 AM Assiati Magana ***@***.***>
wrote:

> ***@***.**** requested changes on this pull request.
> STATUS: Required Changes ♻️♻️♻️ Hi @asadmisrani
> <https://github.com/asadmisrani> , I'm @MaganaAsiati
> <https://github.com/MaganaAsiati>, your reviewer for this PR 👋
>
> Good job so far!
> There are some issues that you still need to work on to go to the next
> project but you are almost there!
>
> [image: so-close-gif-4]
> <https://user-images.githubusercontent.com/95297251/191952218-b7185270-c02b-4e4d-89af-eae77fcd8bea.gif>
> Highlights
>
>    - Descriptive README file ✔️👨🏻‍💻
>    - Great work using the right GitHub Flow.✔️
>    - No Linter errors ✔️
>
> Required Changes ♻️
>
>    -
>
>    You did a great job by setting up correctly linters, but as you see it
>    still showing red color at the stylelint linter because you still have
>    linter errors, so please fix this linter by executing this command at the
>    terminal
>    npx stylelint "**/*.{css,scss}" --fix ,so the linter will be happy for
>    that :smiley:
>    -
>
>    Kindly Add a meaningful description to your pull request in the future
>    which gives more information about what you implement in this milestone
>    heavy_check_mark👨🏻‍💻
>
> *Check the comments under the review.*
> Optional suggestions
>
> *Every comment with the [OPTIONAL] prefix is not crucial enough to stop
> the approval of this PR. However, I strongly recommend you take them into
> account as they can make your code better.*
>
>    - [ ]Please consider using best practices when you write commit
>    messages, they should be capitalized sentences in the imperative form, for
>    example: "Add or Upload all files " instead of "All Files uploaded]" ...
>    etc 👨🏻‍💻
>
> Cheers and Happy coding!👏👏👏
>
> Feel free to leave any questions or comments in the PR thread if something
> is not 100% clear.
> Please, remember to tag me in your question so I can receive the
> notification.
>
> *Please, do not open a new Pull Request for re-reviews. You should use the
> same Pull Request submitted for the first review, either valid or invalid
> unless it is requested otherwise.*
> ------------------------------
>
> *As described in the Code reviews limits policy
> <https://microverse.zendesk.com/hc/en-us/articles/1500004088561> you have a
> limited number of reviews per project (check the exact number in your
> Dashboard). If you think that the code review was not fair, you can request
> a second opinion using this form <https://airtable.com/shrQAqnBwek5a0O0s>.*
>
> ------------------------------
>
> In README.md
> <https://github.com/asadmisrani/microverse.github.io/pull/1#discussion_r1066380052>
> :
>
> > -# microverse.github.io
>
> \ No newline at end of file
>
> +# microverse.github.io
>
> +
>
> +> This is website that used for Microverse
>
> +
>
> +**Built With**
>
> +Html
>
> +Css-3
>
> +Linters
>
>
>
>    - kindly Add a descriptive README file to your project - please use this
>    template <https://github.com/microverseinc/readme-template> as
>    required in the project but customize it to your project. as it was
>    required here
>    <https://github.com/asadmisrani/microverse.github.io/pull/1/files#r1066289861>
>    by the previous reviewer 👍
>
> ------------------------------
>
> In .gitignore
> <https://github.com/asadmisrani/microverse.github.io/pull/1#discussion_r1066385121>
> :
>
> > +yarn-debug.log*
>
> +yarn-error.log*
>
> +lerna-debug.log*
>
> +
>
> +# Diagnostic reports (https://nodejs.org/api/report.html)
>
> +report.[0-9]*.[0-9]*.[0-9]*.[0-9]*.json
>
> +
>
> +# Runtime data
>
> +pids
>
> +*.pid
>
> +*.seed
>
> +*.pid.lock
>
> +
>
> +# Directory for instrumented libs generated by jscoverage/JSCover
>
> +lib-cov
>
> +
>
> +# Coverage directory used by tools like istanbul
>
> +coverage
>
> +*.lcov
>
> +
>
> +# nyc test coverage
>
> +.nyc_output
>
> +
>
> +# Grunt intermediate storage (https://gruntjs.com/creating-plugins#storing-task-files)
>
> +.grunt
>
> +
>
> +# Bower dependency directory (https://bower.io/)
>
> +bower_components
>
> +
>
> +# node-waf configuration
>
> +.lock-wscript
>
> +
>
> +# Compiled binary addons (https://nodejs.org/api/addons.html)
>
> +build/Release
>
> +
>
> +# Dependency directories
>
> +node_modules/
>
> +jspm_packages/
>
> +
>
> +# TypeScript v1 declaration files
>
> +typings/
>
> +
>
> +# TypeScript cache
>
> +*.tsbuildinfo
>
> +
>
> +# Optional npm cache directory
>
> +.npm
>
> +
>
> +# Optional eslint cache
>
> +.eslintcache
>
> +
>
> +# Microbundle cache
>
> +.rpt2_cache/
>
> +.rts2_cache_cjs/
>
> +.rts2_cache_es/
>
> +.rts2_cache_umd/
>
> +
>
> +# Optional REPL history
>
> +.node_repl_history
>
> +
>
> +# Output of 'npm pack'
>
> +*.tgz
>
> +
>
> +# Yarn Integrity file
>
> +.yarn-integrity
>
> +
>
> +# dotenv environment variables file
>
> +.env
>
> +.env.test
>
> +
>
> +# parcel-bundler cache (https://parceljs.org/)
>
> +.cache
>
> +
>
> +# Next.js build output
>
> +.next
>
> +
>
> +# Nuxt.js build / generate output
>
> +.nuxt
>
> +dist
>
> +
>
> +# Gatsby files
>
> +.cache/
>
> +# Comment in the public line in if your project uses Gatsby and *not* Next.js
>
> +# https://nextjs.org/blog/next-9-1#public-directory-support
>
> +# public
>
> +
>
> +# vuepress build output
>
> +.vuepress/dist
>
> +
>
> +# Serverless directories
>
> +.serverless/
>
> +
>
> +# FuseBox cache
>
> +.fusebox/
>
> +
>
> +# DynamoDB Local files
>
> +.dynamodb/
>
> +
>
> +# TernJS port file
>
> +.tern-port
>
>
>
>    - kindly Remove unnecessary files in your gitignore and make sure that
>    the file listing test.md file and node_modules directory is added in
>    the gitignore (as it can be added by accident while setting up Linters). as
>    it shows above alone node_modules is present in the git ignore please
>    add also test.md 👍
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/asadmisrani/microverse.github.io/pull/1#pullrequestreview-1243021556>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AYQGAKFR6VQVGONYWNWWDJ3WRXLJRANCNFSM6AAAAAATXG3C4E>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

asadmisrani commented:
Hello Dear,
The npx stylelint "**/*.{css,scss}" --fix you have sent to me not working
in my terminal and I am still getting same problems in my linter.
[image: error.png]

On Wed, Jan 11, 2023 at 3:33 AM asad shareef ***@***.***>
wrote:

> Hello Sir,
> I have already fixed this error npx stylelint "**/*.{css,scss}" --fix can
> you please approve my Project then I will go further.
> Thank You
> Regards
> Asad Shareef
>
> On Wed, Jan 11, 2023 at 2:56 AM Assiati Magana ***@***.***>
> wrote:
>
>> ***@***.**** requested changes on this pull request.
>> STATUS: Required Changes ♻️♻️♻️ Hi @asadmisrani
>> <https://github.com/asadmisrani> , I'm @MaganaAsiati
>> <https://github.com/MaganaAsiati>, your reviewer for this PR 👋
>>
>> Good job so far!
>> There are some issues that you still need to work on to go to the next
>> project but you are almost there!
>>
>> [image: so-close-gif-4]
>> <https://user-images.githubusercontent.com/95297251/191952218-b7185270-c02b-4e4d-89af-eae77fcd8bea.gif>
>> Highlights
>>
>>    - Descriptive README file ✔️👨🏻‍💻
>>    - Great work using the right GitHub Flow.✔️
>>    - No Linter errors ✔️
>>
>> Required Changes ♻️
>>
>>    -
>>
>>    You did a great job by setting up correctly linters, but as you see
>>    it still showing red color at the stylelint linter because you still have
>>    linter errors, so please fix this linter by executing this command at the
>>    terminal
>>    npx stylelint "**/*.{css,scss}" --fix ,so the linter will be happy
>>    for that :smiley:
>>    -
>>
>>    Kindly Add a meaningful description to your pull request in the
>>    future which gives more information about what you implement in this
>>    milestone heavy_check_mark👨🏻‍💻
>>
>> *Check the comments under the review.*
>> Optional suggestions
>>
>> *Every comment with the [OPTIONAL] prefix is not crucial enough to stop
>> the approval of this PR. However, I strongly recommend you take them into
>> account as they can make your code better.*
>>
>>    - [ ]Please consider using best practices when you write commit
>>    messages, they should be capitalized sentences in the imperative form, for
>>    example: "Add or Upload all files " instead of "All Files uploaded]" ...
>>    etc 👨🏻‍💻
>>
>> Cheers and Happy coding!👏👏👏
>>
>> Feel free to leave any questions or comments in the PR thread if
>> something is not 100% clear.
>> Please, remember to tag me in your question so I can receive the
>> notification.
>>
>> *Please, do not open a new Pull Request for re-reviews. You should use
>> the same Pull Request submitted for the first review, either valid or
>> invalid unless it is requested otherwise.*
>> ------------------------------
>>
>> *As described in the Code reviews limits policy
>> <https://microverse.zendesk.com/hc/en-us/articles/1500004088561> you have a
>> limited number of reviews per project (check the exact number in your
>> Dashboard). If you think that the code review was not fair, you can request
>> a second opinion using this form <https://airtable.com/shrQAqnBwek5a0O0s>.*
>>
>> ------------------------------
>>
>> In README.md
>> <https://github.com/asadmisrani/microverse.github.io/pull/1#discussion_r1066380052>
>> :
>>
>> > -# microverse.github.io
>>
>> \ No newline at end of file
>>
>> +# microverse.github.io
>>
>> +
>>
>> +> This is website that used for Microverse
>>
>> +
>>
>> +**Built With**
>>
>> +Html
>>
>> +Css-3
>>
>> +Linters
>>
>>
>>
>>    - kindly Add a descriptive README file to your project - please use this
>>    template <https://github.com/microverseinc/readme-template> as
>>    required in the project but customize it to your project. as it was
>>    required here
>>    <https://github.com/asadmisrani/microverse.github.io/pull/1/files#r1066289861>
>>    by the previous reviewer 👍
>>
>> ------------------------------
>>
>> In .gitignore
>> <https://github.com/asadmisrani/microverse.github.io/pull/1#discussion_r1066385121>
>> :
>>
>> > +yarn-debug.log*
>>
>> +yarn-error.log*
>>
>> +lerna-debug.log*
>>
>> +
>>
>> +# Diagnostic reports (https://nodejs.org/api/report.html)
>>
>> +report.[0-9]*.[0-9]*.[0-9]*.[0-9]*.json
>>
>> +
>>
>> +# Runtime data
>>
>> +pids
>>
>> +*.pid
>>
>> +*.seed
>>
>> +*.pid.lock
>>
>> +
>>
>> +# Directory for instrumented libs generated by jscoverage/JSCover
>>
>> +lib-cov
>>
>> +
>>
>> +# Coverage directory used by tools like istanbul
>>
>> +coverage
>>
>> +*.lcov
>>
>> +
>>
>> +# nyc test coverage
>>
>> +.nyc_output
>>
>> +
>>
>> +# Grunt intermediate storage (https://gruntjs.com/creating-plugins#storing-task-files)
>>
>> +.grunt
>>
>> +
>>
>> +# Bower dependency directory (https://bower.io/)
>>
>> +bower_components
>>
>> +
>>
>> +# node-waf configuration
>>
>> +.lock-wscript
>>
>> +
>>
>> +# Compiled binary addons (https://nodejs.org/api/addons.html)
>>
>> +build/Release
>>
>> +
>>
>> +# Dependency directories
>>
>> +node_modules/
>>
>> +jspm_packages/
>>
>> +
>>
>> +# TypeScript v1 declaration files
>>
>> +typings/
>>
>> +
>>
>> +# TypeScript cache
>>
>> +*.tsbuildinfo
>>
>> +
>>
>> +# Optional npm cache directory
>>
>> +.npm
>>
>> +
>>
>> +# Optional eslint cache
>>
>> +.eslintcache
>>
>> +
>>
>> +# Microbundle cache
>>
>> +.rpt2_cache/
>>
>> +.rts2_cache_cjs/
>>
>> +.rts2_cache_es/
>>
>> +.rts2_cache_umd/
>>
>> +
>>
>> +# Optional REPL history
>>
>> +.node_repl_history
>>
>> +
>>
>> +# Output of 'npm pack'
>>
>> +*.tgz
>>
>> +
>>
>> +# Yarn Integrity file
>>
>> +.yarn-integrity
>>
>> +
>>
>> +# dotenv environment variables file
>>
>> +.env
>>
>> +.env.test
>>
>> +
>>
>> +# parcel-bundler cache (https://parceljs.org/)
>>
>> +.cache
>>
>> +
>>
>> +# Next.js build output
>>
>> +.next
>>
>> +
>>
>> +# Nuxt.js build / generate output
>>
>> +.nuxt
>>
>> +dist
>>
>> +
>>
>> +# Gatsby files
>>
>> +.cache/
>>
>> +# Comment in the public line in if your project uses Gatsby and *not* Next.js
>>
>> +# https://nextjs.org/blog/next-9-1#public-directory-support
>>
>> +# public
>>
>> +
>>
>> +# vuepress build output
>>
>> +.vuepress/dist
>>
>> +
>>
>> +# Serverless directories
>>
>> +.serverless/
>>
>> +
>>
>> +# FuseBox cache
>>
>> +.fusebox/
>>
>> +
>>
>> +# DynamoDB Local files
>>
>> +.dynamodb/
>>
>> +
>>
>> +# TernJS port file
>>
>> +.tern-port
>>
>>
>>
>>    - kindly Remove unnecessary files in your gitignore and make sure
>>    that the file listing test.md file and node_modules directory is
>>    added in the gitignore (as it can be added by accident while setting up
>>    Linters). as it shows above alone node_modules is present in the git
>>    ignore please add also test.md 👍
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/asadmisrani/microverse.github.io/pull/1#pullrequestreview-1243021556>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AYQGAKFR6VQVGONYWNWWDJ3WRXLJRANCNFSM6AAAAAATXG3C4E>
>> .
>> You are receiving this because you were mentioned.Message ID:
>> ***@***.***>
>>
>

asadmisrani committed:
Linter error has been solved
